java/change-atmosphere-effect/src/main/java/com/esri/arcgisruntime/sample/changeatmosphereeffect/MainActivity.java

```diff
@@ -0,0 +1,113 @@
+/* Copyright 2018 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.esri.arcgisruntime.sample.changeatmosphereeffect;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.Menu;
+import android.view.MenuInflater;
+import android.view.MenuItem;
+
+import com.esri.arcgisruntime.mapping.ArcGISScene;
+import com.esri.arcgisruntime.mapping.ArcGISTiledElevationSource;
+import com.esri.arcgisruntime.mapping.Basemap;
+import com.esri.arcgisruntime.mapping.Surface;
+import com.esri.arcgisruntime.mapping.view.AtmosphereEffect;
+import com.esri.arcgisruntime.mapping.view.SceneView;
+import com.esri.arcgisruntime.mapping.view.Camera;
+
+
+public class MainActivity extends AppCompatActivity {
+
+    private static final String TAG = MainActivity.class.getSimpleName();
+    private SceneView mSceneView;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        // create a scene and add a basemap to it
+        ArcGISScene scene = new ArcGISScene();
+        scene.setBasemap(Basemap.createImagery());
+
+        // create SceneView from layout
+        mSceneView = findViewById(R.id.sceneView);
+        mSceneView.setScene(scene);
+
+        // add base surface for elevation data
+        Surface surface = new Surface();
+        ArcGISTiledElevationSource elevationSource = new ArcGISTiledElevationSource("R.string.elevation_image_service");
```

Comment:
Should be: `new ArcGISTiledElevationSource(getString(R.string.elevation_image_service));`